### PR TITLE
WFCORE-1413 Web console doesn't accept passwords with non-ascii chara…

### PR DESCRIPTION
**Pull request by spyrkob**
…cters

JIRA: https://issues.jboss.org/browse/WFCORE-1413

### CODE REVIEW

**Comment by bstansberry** [comment]
this is ok to test

**Comment by spyrkob** [comment]
retest this please

File: domain-http/interface/src/main/java/org/jboss/as/domain/http/server/ManagementHttpServer.java [hunk 1]
```diff
@@ -113,6 +118,7 @@
     private volatile AcceptingChannel<SslConnection> secureServer;
     private final SSLContext sslContext;
     private final SslClientAuthMode sslClientAuthMode;
+    private static Map<Pattern, Charset> userAgentCharsetMap = generateCharsetMap();
```
Comment (bstansberry) [inline domain-http/interface/src/main/java/org/jboss/as/domain/http/server/ManagementHttpServer.java]: This needs the 'final' keyword.

Also, if you don't mind, I'd prefer that it's declared above the instance fields. That's a minor codestyle nit.

**Comment by bstansberry** [comment]
Please squash the 2 commits when you rebase. Thanks!

**Comment by bstansberry** [comment]
Closing in favor of #1478 (which is just a rebase of this.)

**Pull request closed without merging**